### ISSUE TITLE
Support Drupal Distributions

### ISSUE THREAD

**Issue by mstenta** [docker-library/drupal#64]
I think with a little bit of work, along with the changes I proposed in #63 , we could make it possible for Drupal distribution maintainers to inherit from these images to provide Docker images for their distributions.

Distributions are pre-packaged combinations of Drupal core and other selected modules, themes, etc. Examples include OpenAtrium (https://www.drupal.org/project/openatrium), Commerce Kickstart (https://www.drupal.org/project/commerce_kickstart), and farmOS (https://www.drupal.org/project/farm).

Currently, the Dockerfiles create two environmental variables:

```
ENV DRUPAL_VERSION 7.50
ENV DRUPAL_MD5 f23905b0248d76f0fc8316692cd64753
```

These are used to download and verify the packaged drupal codebase:

```
curl -fSL "http://ftp.drupal.org/files/projects/drupal-${DRUPAL_VERSION}.tar.gz"

echo "${DRUPAL_MD5} /usr/src/drupal-${DRUPAL_VERSION}.tar.gz" | md5sum -c -
```

Drupal distributions can be downloaded and verified using the same pattern, except instead of "drupal" they use their project name (ie: "farm").

As a first step, we can add an additional environmental variable:

```
ENV DRUPAL_PROJECT drupal
```

And use that to build the download URL:

```
curl -fSL "http://ftp.drupal.org/files/projects/${DRUPAL_PROJECT}-${DRUPAL_VERSION}.tar.gz"
```

There are a few other things to figure out... the code changes introduced in #63 automatically compare the "current" and "desired" Drupal versions using Drush. But that only looks at the Drupal core version. A distribution will have it's own version on top of the Drupal core version. So we will need to tweak the way that version comparison code works too.

**Comment by mstenta** [comment]
I closed PR #63 - so this will need to be re-thought.

**Comment by yosifkit** [comment]
Closing old issue.
